zt_backend/models/components/list.py

```diff
@@ -0,0 +1,128 @@
+from pydantic import Field
+from typing import List, Optional, Union
+from zt_backend.models.components.zt_component import ZTComponent
+
+
+class ListComponent(ZTComponent):
+    """List the Primary components holding the items in the scorllable format"""
+
+    component: str = Field("v-list", description="Vue component name for List")
+    childComponents: List[str] = Field(
+        [], description="List of child component ids to be placed within the List"
+    )
+    color: str = Field(None, description="Background color of the List")
+    elevation: int = Field(
+        None,
+        ge=0,
+        le=24,
+        description="Elevation level of the List. Must be between 0 and 24",
+    )
+    density: str = Field(
+        None,
+        enum=["default", "comfortable", "compact"],
+        description="Density of the List",
+    )
+    width: Union[int, str] = Field("100%", description="Width of the List")
+    height: Union[int, str] = Field("100%", description="Height of the List")
+
+
+class ListItem(ZTComponent):
+    """List Item helps define properties of individual items in the list"""
+
+    component: str = Field(
+        "v-list-item", description="Vue component name for List Item"
+    )
+    title: str = Field("", description="item title")
+    color: str = Field(None, description="Background color of the List item")
+    elevation: int = Field(
+        None,
+        ge=0,
+        le=24,
+        description="Elevation level of the List item. Must be between 0 and 24",
+    )
+    density: str = Field(
+        "default",
+        enum=["default", "comfortable", "compact"],
+        description="Density of the List Item: default | comfortable | compact",
+    )
+    width: Union[int, str] = Field("100%", description="Width of the List item")
+    height: Union[int, str] = Field("100%", description="Height of the List item")
+    value: str = Field("", description="value for List Item")
```

Comment:
Is value used on this?

Reply:
Created the attribute as per the props in[ v-list-item ](https://vuetifyjs.com/en/api/v-list-item/#props-value)API Docs

Reply:
What is value used for in this context

Reply:
Following up